sos/plugins/__init__.py

```diff
@@ -521,14 +521,15 @@ def add_copy_spec(self, copyspecs):
             self._log_info("added copyspec '%s'" % copy_paths)
```

Comment:
Commit message is a tiny bit inaccurate: the passed environment is added to `os.environ`, rather than overwriting it entirely (but values from the argument do take precedence).